packages/lit-dev-content/src/icons/check-code-icon.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {html} from 'lit';
+
+// Source: google fonts material icons
```

Comment:
Let's link to the specific icon URL, like https://material.io/resources/icons/?icon=restart_alt.